utils/releaser/src/ReleaseWorker/ReleaseCandidate/SetMutualDependenciesToVersionReleaseWorker.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+use Symplify\MonorepoBuilder\FileSystem\ComposerJsonProvider;
+use Symplify\MonorepoBuilder\InterdependencyUpdater;
+use Symplify\MonorepoBuilder\Package\PackageNamesProvider;
+use Symplify\MonorepoBuilder\Release\Message;
+use Symplify\MonorepoBuilder\Utils\Utils;
+
+final class SetMutualDependenciesToVersionReleaseWorker extends AbstractShopsysReleaseWorker
```

Comment:
From testing - `project-base` is not included (it is not located in `/packages`). Also, the dumped version is not correct - should be `7.0.0-beta3`, however, I see `^7.0` in modified composer.jsons instead

Reply:
The `^7.0` is correct, because you should use branch aliases insted of specific tags.

I've changed that to the exact name of the tag to make you happy :+1:

Reply:
Thank you :smile: :+1: 
I believe the correct flow would be finishing this automation and not changing such things. Afterward, it would be easy to propose such changes in a separate pull request :slightly_smiling_face: